### PR TITLE
code health sha1

### PR DESCRIPTION
**Pull request by pauli-oracle**
- [x] tests are added or updated

 Update sha1test to use the test infrastructure

### CODE REVIEW

File: test/sha1test.c [hunk 1]
```diff
@@ -31,81 +33,75 @@ static char *ret[] = {
...
-            printf("error calculating SHA1 on '%s'\n", test[i]);
-            printf("got %s instead of %s\n", p, *R);
-            err++;
-        } else
-            printf("test %d ok\n", i + 1);
-        R++;
-    }
+    if (!TEST_true(EVP_Digest(test[i], tlen, md, NULL, EVP_sha1(), NULL)))
+        goto end;
+    if (!TEST_str_eq(pt(md), ret[i]))
```
Comment (mattcaswell) [inline test/sha1test.c]: Put these in a single if

**Comment by pauli-oracle** [comment]
Unified iffing done.

**Comment by richsalz** [comment]
+1

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Pull request closed without merging**